dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -2801,7 +2802,7 @@ RtpsUdpDataLink::RtpsWriter::gather_gaps_i(const ReaderInfo_rch& reader,
 void
 RtpsUdpDataLink::RtpsWriter::process_acknack(const RTPS::AckNackSubmessage& acknack,
                                              const RepoId& src_prefix,
-                                             MetaSubmessageVec& meta_submessages)
+                                             MetaSubmessageVec& /*meta_submessages*/)
```

Comment:
Should the parameter be removed?

Reply:
It's invoked via function pointer so we can't change the signature.